hieradata/class/production/jenkins.yaml

```diff
@@ -2,7 +2,7 @@
 govuk_jenkins::config::executors: '8'
 
 govuk_jenkins::job_builder::jobs:
-  - govuk_jenkins::jobs::bouncer_cdn
+  - govuk_jenkins::jobs::bouncer_cdn_redirect
```

Comment:
We should not do this in production just yet, as the jobs are currently still needed
For testing, it might be nice to add them to staging, though (AWS staging to not confuse people)